.maintain/frame-weight-template.hbs

```diff
@@ -64,22 +64,22 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 		{{~#each benchmark.components as |c| ~}}
 		{{~#if (not c.is_used)}}_{{/if}}{{c.name}}: u32, {{/each~}}
 	) -> Weight {
-		({{underscore benchmark.base_weight}} as Weight)
+		Weight::from_ref_time({{underscore benchmark.base_weight}} as RefTimeWeight)
 			{{#each benchmark.component_weight as |cw|}}
 			// Standard Error: {{underscore cw.error}}
-			.saturating_add(({{underscore cw.slope}} as Weight).saturating_mul({{cw.name}} as Weight))
+			.saturating_add(Weight::from_ref_time({{underscore cw.slope}} as RefTimeWeight).scalar_saturating_mul({{cw.name}} as RefTimeWeight))
 			{{/each}}
 			{{#if (ne benchmark.base_reads "0")}}
-			.saturating_add(T::DbWeight::get().reads({{benchmark.base_reads}} as Weight))
+			.saturating_add(T::DbWeight::get().reads({{benchmark.base_reads}} as RefTimeWeight))
```

Comment:
Wait a minute, this type casting never made any sense. `base_reads` isn't a unit of weight, but simply denotes the number of reads. There really shouldn't be a type cast to `RefTimeWeight` here.

Reply:
Same with the other similar type casts in this file.

Reply:
Base reads is u32, it needs to be u64 or whatever type `RefTimeWeight` is.

Reply:
i will remove ref_time and this will be just u64 in a follow up pr